server/game/cards/06-CotE/HonorableChallenger.js

```diff
@@ -0,0 +1,39 @@
+const DrawCard = require('../../drawcard.js');
+const { Players, CardTypes } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl');
+const GameActions = require('../../GameActions/GameActions');
+
+class HonorableChallenger extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Initiate a military duel',
+            condition: () => this.isParticipating(),
+            target: {
+                cardType: CardTypes.Character,
+                controller: Players.Opponent,
+                cardCondition: card => card.isParticipating(),
+                gameAction: AbilityDsl.actions.duel(context => ({
+                    type: 'military',
+                    challenger: context.source,
+                    resolutionHandler: (winner) => this.resolutionHandler(context, winner)
+                }))
+            }
+        });
+    }
+
+    resolutionHandler(context, winner) {
+        this.game.addMessage('{0} wins the duel and will not bow as a result of this conflict\'s resolution', winner);
+
+        let doesNotBowEffect = GameActions.cardLastingEffect({
+            match: card => card === winner,
+            targetController: Players.Any,
+            effect: [AbilityDsl.effects.doesNotBow()]
```

Comment:
You shouldn't need `match` or `targetController` here.  You also don't need the effect to be an array, but that's style and readability rather than being objectively wrong